Sources/FoundationInternationalization/Formatting/Date/ICUDateFormatter.swift

```diff
@@ -243,24 +238,16 @@ final class ICUDateFormatter {
     }
 
     static func cachedFormatter(for format: Date.FormatStyle) -> ICUDateFormatter {
-        let hourCycleOption: ICUPatternGenerator.HourCycleOption
```

Comment:
The job of resolving the hour cycle option is sunk to `ICUPatternGenerator.localizedPattern(symbols:locale:calendar:)`